EDU-15949 - Category cover: Business model implementation (#2754)
vtexdocs/dev-portal-content: docs/guides/Business-models/business-models-overview.md

---
title: "Business models"
slug: "business-models-overview"
hidden: false
excerpt: "Choose the VTEX business model that fits your operation."
createdAt: "2026-06-16T00:00:00.000Z"
---

> Your business model defines how your VTEX operation is structured, how catalogs, prices, languages, and integrations are managed, and how your store connects to other markets and sellers.

Reviewer: Mudar de voz passiva para ativa.
```suggestion
Your business model determines how to structure your VTEX operation, manage catalogs, prices, languages, and integrations, and connect your store to other markets and sellers.
```

* Connect two VTEX stores.
* Sell your VTEX store products through a non-VTEX marketplace.
* Onboard non-VTEX sellers into your VTEX marketplace.

The overview guide helps you compare these scenarios and choose the next guide based on the role your store plays in the integration.

For more information, see [Implementing marketplaces](https://developers.vtex.com/docs/guides/implementing-marketplaces).

## Guides in this section

<Flex>

  <WhatsNextCard
    title="Implementing internationalization"
    description="Learn how to prepare your store to operate in multiple languages and locales."
    linkTo="/docs/guides/implementing-internationalization"
    linkTitle="See more"
  />

  <WhatsNextCard
    title="Implementing cross-border stores"
    description="Learn how to structure and start a cross-border operation on VTEX."
    linkTo="/docs/guides/implementing-cross-border-stores"
    linkTitle="See more"
  />

  <WhatsNextCard
    title="Implementing marketplaces"
    description="Compare marketplace implementation scenarios and choose the right integration path."
    linkTo="/docs/guides/implementing-marketplaces"
    linkTitle="See more"
  />

> </Flex>

Author: ```suggestion
</Flex>

```
